ui/components/table/table.go

```diff
@@ -209,6 +219,10 @@ func (m *Model) renderBody() string {
 		Height(m.dimensions.Height).
 		MaxWidth(m.dimensions.Width)
 
+	if m.isLoading {
+		return bodyStyle.Render(m.loadingMessage)
+	}
```

Comment:
If you're up for it, let's make a bit more fancy.
You can add a padding of 1 to the left (I think) and it should center it better.
Also, you can add a spinner to it's left with 	"github.com/charmbracelet/bubbles/spinner". You can see an example in https://github.com/charmbracelet/bubbletea/tree/master/examples/spinner

Reply:
Thanks for taking time for this PR!
Wow, this spinner is great! I'll try to do it!

Reply:
@dlvhdr 

I tried in c2f240565343.
But the spinner does not work.
I will try to fix later. If you know how to fix, please let me know.

https://github.com/dlvhdr/gh-dash/assets/49891479/83fc5747-3a29-4a6e-b0de-be04bbda0386

Reply:
> I tried in https://github.com/dlvhdr/gh-dash/commit/c2f2405653436976519efb6d8f3d649cc8033b43.
But the spinner does not work.
I will try to fix later.
If you have an idea please let me know.

@dlvhdr 
This problem has been fixed.
I added `loadingSpinner, loadingSpinnerCmd := m.Table.LoadingSpinner.Update(msg)` only to `issuesection.go`.
I have to add it to `prsection.go` as well.

After I will been done with TODOs and review the code sanity, I will ask you to review.

Reply:
@dlvhdr 
I'm done with fixing.
Could you review this PR please?